Chapters/1-7-Chapter7.md

```diff
@@ -0,0 +1,493 @@
+
+"And what makes you think we can do that?"
+
+"You think we built a space rail with dumb luck? It took 5 ma'tova of collaboration with your observers. We learned everything we could from them and then we made breakthroughs with our knowledge of electricity and metals. We sent countless test ships into space. Many of them orbited and then softly landed on their own with timers. We had space travel down to a science and that was taken from all of us when Heiko backed out of the deal."
+
+"And you're still bitter about it."
+
+"Damn right I am. There is so much we still don't know about space but we were stopped when we had so much potential. Nobody knows what's up there, in the great beyond. I came to Heiko to work with your observers, but they just wanted to track the same information they had been tracking for ka'tova. Now Palinan is powerless. We're huddling in the dark with no plan but to wait it out."
+
+"So you're just doing this to prove you were right all along. And I'm your Heikoi hero for all the PR."
+
+"Liffa, none of our feelings or policies or quarrels matter if we're all dead. I know that our project is the only way to survive, and I'm using what skills I have to recruit the people we need."
+
+The argumentation side of my brain went quiet. He was right. Even if we were enemies now, it wouldn't matter once we're all freezing and starving to death.
+
+"Is it dangerous?" I asked.
+
+"Of course it is"
+
+"...Does it hurt?"
+
+"What, space?" he laughed. "I suppose you haven't learned what it's like. Imagine a place with no ground; no trees no ocean; no up or down, no sound at all. Just you, Ara, and infinite darkness. That's space."
+
+I struggled to comprehend what that was like, but I didn't have much time until the door opened again.
+
+"Fajro, how are you holding up," the magistrate asked.
+
+"Fine, sir, fine."
+
+He went over to Fajro to check on his head.
+
+"You still look a bit pale, but you're getting better. Let me finish with Liffa, and then we'll get you somewhere we can look closer at your injuries."
+
+"Thank you, magistrate."
+
+The magistrate stood up and moved closer to me.
+
+"Liffa, I realized that I might have been a bit too hasty in my decision to switch the sentence. I've decided to let you choose where to go. Does that sound fair?"
+
+I answered, "Yes, magistrate. It's a generous offer."
+
+"In that case, I'll continue where I left off."
+
+He re-assumed his regal pose with paper and charcoal in hand, and cleared his throat in preparation.
+
+"Liffa Um Heiko. I find you guilty of interruption of the senate during its deliberation. The sentence shall be a period of three la'tova on Ulukoma _or_ a period of one la'tova under the command of Fajro Um Holrag. Do you understand?"
+
+"I do."
+
+"Then I await your decision within the next tova'la. I will be back to -"
+
+"Magistrate, I've already made my decision."
+
+"You have?"
+
+_I'm sorry, Melna._
+
+"I'm going to Holrag."
```

Comment:
Well, I suppose Kip will just have to use his knowledge of biology to stay alive in the wastelands.